src/Advection/positivity_preserving_tracer_advection_operators.jl

```diff
@@ -0,0 +1,89 @@
+
+const ω̂₁ = 5/18 
+const ω̂ₙ = 5/18  
+
+const ε₂ = 1e-20
+
+# Here in the future we can easily add UpwindBiasedFifthOrder 
+const BoundPreservingScheme = PositiveWENO
+
+@inline function div_Uc(i, j, k, grid, advection::BoundPreservingScheme, U, c)
+
+    div_x = bounded_tracer_flux_divergence_x(i, j, k, grid, advection, U.u, c)
+    div_y = bounded_tracer_flux_divergence_y(i, j, k, grid, advection, U.v, c)
+    div_z = bounded_tracer_flux_divergence_z(i, j, k, grid, advection, U.w, c)
+
+    return 1/Vᶜᶜᶜ(i, j, k, grid) * (div_x + div_y + div_z)
+end
+
+@inline function bounded_tracer_flux_divergence_x(i, j, k, grid, advection::BoundPreservingScheme, u, c)
```

Comment:
This looks great!  It seems like these three funcionts are almost identical except for u -> v -> w.  Is that true?

If yes, I wonder if there is away to merge these to one, rather than having three functions that do the same thing?